docs/release-notes/Percona Everest 0.9.1 (2024-04-02).md

```diff
@@ -0,0 +1,27 @@
+# What's new in Percona Everest 0.9.1
+
+To begin your journey with Percona Everest, check out the [Quickstart Guide for Percona Everest](../quickstart-guide/quick-install.md).
+
+Percona Everest is an open source cloud native database platform that helps developers deploy code faster, scale deployments rapidly, and reduce database administration overhead. Plus, you can regain control over your data, database configuration, and DBaaS costs.
+
+Version 0.9.1 introduces the following changes:
+
+
+
+## Release highlights
+
+!!! warning
+    Percona Everest introduces a breaking change that prevents you from directly upgrading to version 0.9.1.
+
+To install Percona Everest 0.9.1, make sure to [uninstall](https://docs.percona.com/everest/install/uninstallEverest.html) any previous versions by running the command:
+
+    everestctl uninstall
```

Comment:
@recharte should we have this warning here too? Looks like we should, just wanted to doublecheck

Reply:
Yes, we should have it. Since the release was only out for 1 day, most users will be coming from 0.8 so we should keep it.

Reply:
@oksana-grishchenko what warning do we wanna inlcude?